### PR TITLE
docs: Remove WalletConnect reference

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
WalletConnect has been deprecated. 

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/guides/build-onchain-apps.mdx [hunk 1]
```diff
@@ -31,8 +31,6 @@ Play with it live [here](https://onchain-app-template.vercel.app).
 
 ## Getting Started
 
-::::steps
-
```
Comment (alessey) [inline site/docs/pages/guides/build-onchain-apps.mdx:35]: is this intentional?

Comment (cpcramer) [reply site/docs/pages/guides/build-onchain-apps.mdx:35]: Screenshot 2024-12-18 at 9 28 39 AM

Comment (alessey) [reply site/docs/pages/guides/build-onchain-apps.mdx:35]: that's odd, it should correspond with :::: at line 92, shows correctly currently https://onchainkit.xyz/guides/build-onchain-apps

**Review by alessey — APPROVED** [review]

**Review by dschlabach — APPROVED** [review]

**Pull request merged**